core/objects.go

```diff
@@ -563,3 +563,15 @@ type FQDNSet struct {
 	Issued  time.Time
 	Expires time.Time
 }
+
+// Order represents the request object that forms the basis of the v2 style
+// issuance flow
+type Order struct {
```

Comment:
[Section 7.1.3 of draft-07](https://tools.ietf.org/html/draft-ietf-acme-acme-07#section-7.1.3) shows a required Status field on Order:
>    status (required, string):  The status of this order.  Possible values are: "pending", "processing", "valid", and "invalid".